source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -2373,6 +2383,33 @@ monitoring can also proactively detect state changes
 (primary stepdown, server becoming unavailable)
 that would otherwise cause future errors.
 
+Why is the driver required to run SDAM flow if server description has not changed?
```

Comment:
I'm not sure we need this rational if we're already following the guidance in "Updating the TopologyDescription", the unit test should be enough to validate the behavior.

Reply:
I find it helpful, and the rationale as I understand it helps explain why the spec prescribes certain behavior or why a test exists which is what I believe is happening here. I do agree that the behavior is verified by the unit test but this seems orthogonal to the purpose of a rationale.

Reply:
I don't think we should set precedent for adding a rationale section for every scenario we can conjure up. For example, we don't have a rationale section for why we remove servers that are `me` mismatched. 

If you think the prose is helpful to better understanding the test then I encourage you to add it inline in the YAML file as comments, helping the test reader to better understand why the test exists.

Reply:
This change is no longer part of this PR.